core/storage/suggestion/gae_models.py

```diff
@@ -402,9 +402,11 @@ def get_user_voiceover_applications(cls, author_id, status=None):
         """
         if status in STATUS_CHOICES:
             return cls.query(ndb.AND(
-                cls.author_id == author_id, cls.status == status)).fetch()
+                cls.author_id == author_id, cls.status == status)).order(
```

Comment:
I know this was there before, but just to check, do you actually need the ndb.AND here? I thought it was possible to use multiple filters in .query().

Also -- tests to verify the ordering? There should be a docstring update too.

Reply:
Added test in the handler PR!